ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/rest/RestTestExecutor.java

```diff
@@ -768,12 +777,19 @@ private static void matchResponseFields(
           assertThat("Response mismatch at " + newPath,
               (String) actualValue, startsWith((String) expectedValue));
         } else {
-          assertThat("Response mismatch at " + newPath, actualValue, is(expectedValue));
+          incrementKey(actualRecords, actualValue.toString());
+          incrementKey(expectedRecords, expectedValue.toString());
         }
       }
     }
   }
 
+  private static void incrementKey(final HashMap<String, Integer> records, final String key){
+    int val = records.containsKey(key) ? records.get(key) : 0;
```

Comment:
Nit: 
```
records.putIfAbsent(key,0);
records.put(key,  map.get(key) + 1);
```
Apparently, you can do this in one line as well:
```
map.merge(records, 1, Integer::sum);
```

Reply:
Nice one liner!